flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisConfigOptions.java

```diff
@@ -298,6 +298,12 @@ public class DorisConfigOptions {
                     .defaultValue(false)
                     .withDescription("Whether to use buffer cache for breakpoint resume");
 
+    public static final ConfigOption<Double> MONGO_CDC_CREATE_SAMPLE_PERCENT =
+            ConfigOptions.key("mongo-cdc.create-sample-percent")
+                    .doubleType()
+                    .defaultValue(0.2)
+                    .withDescription("mongo cdc sample percent");
+
```

Comment:
Would it be better to pass this parameter into cdctools, because it is not used elsewhere?